scripts/bench.sh

```diff
@@ -5,12 +5,37 @@ MYDIR=$(realpath "$(dirname "$0")")
 cd "$MYDIR"
 cd ..
 
+# Optimize contract's wasm binary by crate name.
+opt_wasm () {
+  local CONTRACT_CRATE_NAME=$1
+  local CONTRACT_BIN_NAME="${CONTRACT_CRATE_NAME//-/_}.wasm"
+  local CONTRACT_OPT_BIN_NAME="${CONTRACT_CRATE_NAME//-/_}_opt.wasm"
+
+  echo
+  echo "Optimising $CONTRACT_CRATE_NAME WASM binary"
+  # https://rustwasm.github.io/book/reference/code-size.html#use-the-wasm-opt-tool
+  wasm-opt -O3 -o ./target/wasm32-unknown-unknown/release/"$CONTRACT_OPT_BIN_NAME" ./target/wasm32-unknown-unknown/release/"$CONTRACT_BIN_NAME"
+}
+
+# Retrieve all alphanumeric contract's crate names in `./examples` directory.
+get_example_crate_names () {
+  # shellcheck disable=SC2038
+  # NOTE: optimistically relying on the 'name = ' string at Cargo.toml file
+  find ./examples -maxdepth 2 -type f -name "Cargo.toml" | xargs grep 'name = ' | grep -oE '".*"' | tr -d "'\""
+}
+
 cargo build --release --target wasm32-unknown-unknown -Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort
 
+# Optimize contract's wasm for gas usage.
```

Comment:
Previously you have `Optimising`, here `Optimize`. Use either AmE or BrE version please.